developers/weaviate/model-providers/friendliai/generative.md

```diff
@@ -0,0 +1,255 @@
+
+ <TabItem value="py" label="Python API v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START SinglePromptExample"
+      endMarker="# END SinglePromptExample"
+      language="py"
+    />
+  </TabItem>
+
+ <TabItem value="js" label="JS/TS API v3">
+    <FilteredTextBlock
+      text={TSCode}
+      startMarker="// START SinglePromptExample"
+      endMarker="// END SinglePromptExample"
+      language="ts"
+    />
+  </TabItem>
+
+</Tabs>
+
+### Grouped task
+
+![Grouped task RAG integration generates one output for the set of search results](../_includes/integration_friendliai_rag_grouped.png)
+
+To generate one text for the entire set of search results, use the grouped task method.
+
+In other words, when you have `n` search results, the generative model generates one output for the entire group.
+
+<Tabs groupId="languages">
+
+ <TabItem value="py" label="Python API v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START GroupedTaskExample"
+      endMarker="# END GroupedTaskExample"
+      language="py"
+    />
+  </TabItem>
+
+ <TabItem value="js" label="JS/TS API v3">
+    <FilteredTextBlock
+      text={TSCode}
+      startMarker="// START GroupedTaskExample"
+      endMarker="// END GroupedTaskExample"
+      language="ts"
+    />
+  </TabItem>
+
+</Tabs>
+
+## References
+
+### Generative parameters
+
+<!-- Zain: Any parameters discuss here? Pls see e.g. Cohere/OpenAI page for comparison -->
+
+Configure the following generative parameters to customize the model behavior.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python API v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START FullGenerativeFriendliAI"
+      endMarker="# END FullGenerativeFriendliAI"
+      language="py"
+    />
+  </TabItem>
+
+  <TabItem value="js" label="JS/TS API v3">
+    <FilteredTextBlock
+      text={TSCode}
+      startMarker="// START FullGenerativeFriendliAI"
+      endMarker="// END FullGenerativeFriendliAI"
+      language="ts"
+    />
+  </TabItem>
+
+</Tabs>
+
+For further details on model parameters, see the [FriendliAI API documentation](https://docs.friendli.ai/openapi/create-chat-completions).
+
+### Available models
+
+* `meta-llama-3-8b-instruct`
+* `meta-llama-3-70b-instruct`
+* `mixtral-8x7b-instruct-v0-1`
+
+## Further resources
+
+### Code examples
+
+Once the integrations are configured at the collection, the data management and search operations in Weaviate work identically to any other collection. See the following model-agnostic examples:
+
+- The [how-to: manage data](../../manage-data/index.md) guides show how to perform data operations (i.e. create, update, delete).
+- The [how-to: search](../../search/index.md) guides show how to perform search operations (i.e. vector, keyword, hybrid) as well as retrieval augmented generation.
+
+### References
+
+- FriendliAI [API documentation](https://docs.friendli.ai/openapi/create-chat-completions)
```

Comment:
p99) In line 232, link is set in `[FriendliAI API documentation]`.
But in line 251, link is set in `FriendliAI [API documentation]`.

Reply:
Thanks for the finding 🙏